packages/next/build/webpack-config.ts

```diff
@@ -1034,6 +1034,8 @@ export default async function getBaseWebpackConfig(
                 dependency: { not: ['url'] },
                 options: {
                   isServer,
+                  isDev: dev,
+                  configImageSizes: config.images.imageSizes,
```

Comment:
Doesn't seem like this line is used in the loader

Reply:
Nice catch, I’ll remove it. 

Originally I was thinking of using the smallest size defined in the config but then realized that will yield very different results depending on the config so I just special cases the API instead.